website/pages/api-docs/agent.mdx

```diff
@@ -506,8 +506,8 @@ This endpoint streams logs from the local agent until the connection is closed
 | `GET`  | `/agent/monitor` | `application/json` |
 
 The table below shows this endpoint's support for
-[blocking queries](/api#blocking-queries) and
-[required ACLs](/api#acls).
+[blocking queries](/api-docs#blocking-queries) and
+[required ACLs](/api-docs#acls).
```

Comment:
It may be worth separating this `/api` -> `/api-docs` change into another PR since it's so simple but so wide-reaching. Also while nice, it isn't really necessary -- `/api` redirects to `/api-docs` already.